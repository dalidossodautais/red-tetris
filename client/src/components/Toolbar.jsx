import styled from "styled-components";
import Toolbar from "@material-ui/core/Toolbar";

export default styled(Toolbar).attrs((props) => ({
  backgroundcolor: props.theme.primaryBackgroundColor,
}))`
  background-color: ${(props) => props.backgroundcolor};
  margin-bottom: ${(props) => props.marginbottom};
`;
