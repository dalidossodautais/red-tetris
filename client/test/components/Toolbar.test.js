import React from "react";
import renderer from "react-test-renderer";
import Toolbar from "../../src/components/Toolbar";

describe("Toolbar", () => {
  test("Standard", () => {
    const component = renderer.create(<Toolbar />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
