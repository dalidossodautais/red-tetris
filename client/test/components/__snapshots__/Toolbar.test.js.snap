// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar Standard 1`] = `
<div
  className="MuiToolbar-root MuiToolbar-regular sc-bdfBwQ bRqnyM MuiToolbar-gutters"
/>
`;

exports[`Toolbar Standard 2`] = `
<div
  className="MuiToolbar-root MuiToolbar-regular sc-bdfBwQ bRqnyM MuiToolbar-gutters"
/>
`;
